Makefile

```diff
@@ -8,3 +8,7 @@ rust:
 
 binlog:
 	./generate-binlog.sh
+
+c++:
+	./generate-cpp.sh
+	rm -rf build_cpp && mkdir build_cpp && cd build_cpp && cmake ../cpp && make && cd .. && rm -rf build_cpp
```

Comment:
Why not put this line in generate-cpp.sh ?

Reply:
done